src/waveletTypes.jl

```diff
@@ -25,11 +25,11 @@ struct Morse <: ContinuousWavelets.ContWaveClass
     cf::Float64
 end
 
-N = 100; ga = 3; be = 1; cf = 1;
+ga = 3; be = 1; cf = 1;
```

Comment:
So I get that you're using these in the next line, but it's better to not have these variables floating around in the ContinuousWavelets namespace, which is what this will do. Please just include them directly in the next line. Also, I suspect that b/c the above are floats and these are ints, it's causing errors in the tests. You may want to define a method for Morse that converts its input into floats.

Reply:
I don't know how to convert it directly in Struct, so I use the Morse_convert function for conversion of datatype.